app/code/community/Billie/Core/etc/config.xml

```diff
@@ -2,7 +2,7 @@
 <config>
     <modules>
         <Billie_Core>
-            <version>0.1.7</version>
+            <version>0.2.3</version>
```

Comment:
This will break the sql installations, since the last version we have is 0.1.7, and I dont see in these commits newer versions.